starter-code/src/viking.js

```diff
@@ -1,11 +1,92 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+  this.attack = function () {
```

Comment:
Assigning the attack function to constructor function will get you to pass the test but the proper way would be using the prototypal inheritance which would be something like this:
```
Soldier.prototype.attack = function() {
  return this.strength;
};
```